File: code/plot_energies.py
#!/usr/bin/env python

#
# PYTHON SCRIPT FOR PLOTTING THE ENERGY LEVELS, USING MATPLOTLIB
# Made by Simon Nilsson 2016-11-10
#


import matplotlib.pylab as plt
import numpy as np

filename = "energies.dat"


data = np.loadtxt(filename)

plt.figure(figsize=(8,6))
plt.plot(data[:,0], data[:,1], '-')

# Set labels
plt.xlabel('Time / [ps]')
plt.ylabel('Energy / [eV]')
plt.title('$E(t)$')


# Axis limits
#plt.xlim([0, 100])

# Tick fontsize
plt.xticks(fontisize=12)
plt.yticks(fontsize=12)


# Save and display the plot
plt.savefig('energies.pdf')
plt.show()

File: code/Task7/plot_energies.m
close % plot the energies
% Created by Martin Gren 2014-10-25.

% load the data file
data = importdata('energy.dat');

%plot 
figure;
plot(data(:,1),data(:,2)+data(:,3),'-')
hold on
plot(data(:,1),data(:,2),'-')
plot(data(:,1),data(:,3),'-')

% labels
xlabel('Time / [ASU]');
ylabel('Energy / [ASU]');

% legend
legend('Total energy','Potential energy','Kinetic energy');

title('Awesome title')